pip-requires.txt

```diff
@@ -1,35 +0,0 @@
--e git+https://github.com/nyaruka/smartmin.git#egg=smartmin
```

Comment:
Every single thing in this file is in `pip-freeze.txt` - why on earth would you have them twice?

You even install this verison in your `Dockerfile` first and then overwrite all the latest versions with pinned versions from `pip-freeze.txt`.

Madness.